operator/src/main/java/org/keycloak/operator/v2alpha1/KeycloakDeployment.java

```diff
@@ -101,60 +99,6 @@ private Deployment fetchExistingDeployment() {
                 .get();
     }
 
-    private void addInitContainer(Deployment baseDeployment, List<String> extensions) {
-        var skipExtensions = Optional
-                .ofNullable(extensions)
-                .map(e -> e.isEmpty())
-                .orElse(true);
-
-        if (skipExtensions) {
-            return;
-        }
-
-        // Add emptyDir Volume
-        var volumes = baseDeployment.getSpec().getTemplate().getSpec().getVolumes();
-
-        var extensionVolume = new VolumeBuilder()
-                .withName(Constants.EXTENSIONS_VOLUME_NAME)
-                .withNewEmptyDir()
-                .endEmptyDir()
-                .build();
-
-        volumes.add(extensionVolume);
-        baseDeployment.getSpec().getTemplate().getSpec().setVolumes(volumes);
-
-        // Add the main deployment Volume Mount
-        var container = baseDeployment.getSpec().getTemplate().getSpec().getContainers().get(0);
-        var containerVolumeMounts = container.getVolumeMounts();
-
-        var extensionVM = new VolumeMountBuilder()
-                .withName(Constants.EXTENSIONS_VOLUME_NAME)
-                .withMountPath(Constants.KEYCLOAK_PROVIDERS_FOLDER)
```

Comment:
Please remove the unused constants as well:
https://github.com/keycloak/keycloak/blob/1f10a4feb2dfea88bb62e76e314c4e999ef75cc9/operator/src/main/java/org/keycloak/operator/Constants.java#L41-L46

Reply:
👍  done